src/System.Buffers.Primitives/System/Buffers/ReadOnlyBuffer_helpers.cs

```diff
@@ -10,8 +10,65 @@ namespace System.Buffers
 {
     public readonly partial struct ReadOnlyBuffer 
     {
+        public static int Seek(Position begin, Position end, out Position result, byte byte0)
```

Comment:
Change them to be extension/instance methods on `ReadOnlyBuffer`

Reply:
Let's do it separately. I think I have seen some code that calls the APIs with positions other that the buffer's start/end

Reply:
I think `Seek` is the last one, but we can do it in a separate pass, just don't like moving things back and forth.